### ISSUE TITLE
AttributeError: 'Attribute' object has no attribute 'id'

### ISSUE THREAD

**Issue by nijel** [rocioar/flake8-django#62]
This seems similar to https://github.com/rocioar/flake8-django/issues/48, but the exception is raised somewhere else:

```pytb
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/multiprocessing/pool.py", line 125, in worker
    result = (True, func(*args, **kwds))
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/multiprocessing/pool.py", line 48, in mapstar
    return list(map(*args))
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8/checker.py", line 666, in _run_checks
    return checker.run_checks()
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8/checker.py", line 598, in run_checks
    self.run_ast_checks()
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8/checker.py", line 502, in run_ast_checks
    for (line_number, offset, text, check) in runner:
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8_django/checker.py", line 66, in run
    parser.visit(self.tree)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/ast.py", line 360, in visit
    return visitor(node)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/ast.py", line 368, in generic_visit
    self.visit(item)
  File "/opt/hostedtoolcache/Python/3.8.2/x64/lib/python3.8/ast.py", line 360, in visit
    return visitor(node)
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8_django/checker.py", line 49, in visit_ClassDef
    self.capture_issues_visitor('ClassDef', node)
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8_django/checker.py", line 42, in capture_issues_visitor
    self.issues.extend(issues)
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8_django/checkers/model_content_order.py", line 88, in get_issues
    element_type = self.get_element_type(element)
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8_django/checkers/model_content_order.py", line 104, in get_element_type
    if check(element):
  File "/home/runner/.cache/pre-commit/repo6ciqdvs1/py_env-python3.8/lib/python3.8/site-packages/flake8_django/checkers/model_content_order.py", line 26, in is_assignment_call
    return isinstance(node, Assign) and isinstance(node.value, Call) and not node.targets[0].id.isupper()
AttributeError: 'Attribute' object has no attribute 'id'
```

Probably similar patch is needed.

**Comment by rocioar** [comment]
Thanks for the heads-up!  Do you have the example code in which it is failing?

**Comment by nijel** [comment]
How to easily figure that out what is causing problems? I noticed the failres in pre-commit based CI pipeline just after flake8-django 1.0.0 was released (https://github.com/WeblateOrg/weblate/runs/601100832?check_suite_focus=true)

**Comment by rocioar** [comment]
There is no easy way actually. I will try to find this one, no worries.

**Comment by nijel** [comment]
This is code that triggers it:

```python
from django.contrib.auth import User
from django.db import models
from django.utils.translation import gettext_lazy as _


class AuditLog(models.Model):
    """User audit log storage."""

    user = models.ForeignKey(User, on_delete=models.deletion.CASCADE)

    def get_message(self):
        return ""

    get_message.short_description = _("Account activity")
```

The problematic bit is the `short_description` thing, which is used by the Django admin interface.

**Comment by rocioar** [comment]
Awesome, thank you again!
